plugins/social_network/social_network.cpp

```diff
@@ -295,4 +650,15 @@ namespace golos { namespace plugins { namespace social_network {
         });
     }
 
+    void fill_comment_content(const golos::chain::database& db, const comment_object& co, comment_api_object& con) {
+        if (!db.has_index<comment_content_index>()) {
+            return;
+        }
+        auto & content = db.get<comment_content_object, by_comment>(co.id);
+
+        con.title = std::string(content.title.begin(), content.title.end());
+        con.body = std::string(content.body.begin(), content.body.end());
+        con.json_metadata = std::string(content.json_metadata.begin(), content.json_metadata.end());
```

Comment:
Where is root_title?

Reply:
`comment_api_object` doesn't have such field. This is field from `api::discussion`.

Reply:
Add it to comment_api_object. 
But don't add to FC_REFLECT, and will be work fine.